impl/src/error.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fmt;
+
+pub const PLEASE_FILE_A_BUG: &str =
+"Please file an issue at github.com/google/cargo-raze with details.";
+
+#[derive(Debug)]
+pub enum RazeError {
+Generic(String),
```

Comment:
Is this indented correctly?

Reply:
Interesting, I thought I ran `rustfmt` here. I'm not sure how that indentation was created.